sensor/common/heritage/mock.go

```diff
@@ -0,0 +1,15 @@
+package heritage
+
+import "context"
+
+type MockData struct {
+	Data []*PastSensor
+}
```

Comment:
nit
```suggestion
type MockData []*PastSensor
```

Reply:
I personally don't like type aliases and prefer to use full types. Why this is better than the `type MockData struct`?